Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -311,6 +331,12 @@
 upgradeProperties[2017]['2021DesignPU'] = deepcopy(upgradeProperties[2017]['2021Design'])
 upgradeProperties[2017]['2021DesignPU']['ScenToRun'] = ['GenSimFull','DigiFullPU','RecoFullPU','HARVESTFullPU']
 
+upgradeProperties[2017]['2023PU'] = deepcopy(upgradeProperties[2017]['2023'])
+upgradeProperties[2017]['2023PU']['ScenToRun'] = ['GenSimFull','DigiFullPU','RecoFullPU','HARVESTFullPU','NanoFull']
+
+upgradeProperties[2017]['2024PU'] = deepcopy(upgradeProperties[2017]['2024'])
+upgradeProperties[2017]['2024PU']['ScenToRun'] = ['GenSimFull','DigiFullPU','RecoFullPU','HARVESTFullPU','NanoFull']
+
 upgradeProperties[2023] = {
```

Comment:
Will the Run 4 workflows be renamed at some point? Now there would be workflows with "2023" in their names for Run 3 and Run 4.

Reply:
Yes, this will be done as part of #26938. Maybe it's better to hold this until I can implement all the cleanup there.

Reply:
@kpedro88 @makortel Nominally Run 3 will end after 2023 but there are studies underway to investigate detector performance if Run 3 is extended by one year. The "2023" and "2024" scenarios correspond to simulation of the CMS detector after integrated luminosities of 450 fb<sup>-1</sup> and 550 fb<sup>-1</sup> respectively. See slide 18 of the [PPD update at CMS Week](https://indico.cern.ch/event/825245/contributions/3463951/attachments/1867648/3071883/19-06-24_PPD_CMSWeek.pdf)

From the AlCa perspective, we would like to introduce 11_0_X queues for the 450 fb<sup>-1</sup> and 550 fb<sup>-1</sup> scenarios so that we can integrate new developments for these queues while closing the 10_6_X mcRun3_2023 and mcRun3_2024 queues to anything other than bug fixes. At the same time, we would like to rename all of the Run 3 and later GTs to standardize the naming. The year would be left off the name of the "mcRun4" scenario until a need to distinguish between the various years arises. This is the content of the [autoCond update](https://github.com/cms-sw/cmssw/compare/CMSSW_11_0_X...christopheralanwest:alca-gt-2023-2024?expand=1) that @zhenhu referenced at the introduction to this PR. 

@kpedro88, would the autoCond update by itself conflict with your goals in #26938?

Reply:
Currently the autoCond key used in the Phase 2 workflows is `auto:phase2_realistic`. If this key is preserved (even if the value changes), it should not interfere with the upcoming Phase 2 cleanup.

Reply:
@kpedro88 I think we need to agree on a global strategy before integrating bits and pieces that contradict each other

Reply:
I would prefer to submit the phase 2 cleanup before this, as it has been delayed for several weeks now by other conflicting developments. Today is a holiday in the US, but I might be able to prepare it tomorrow.